README.md

```diff
@@ -1 +1,166 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  
+- [👥 Authors](#authors)
+
+- 🔭 [Future Feature](#futurefeatures)
+
+
+- 🤝 [Contributing](#contributing)
+
+- ⭐️ [Show your support](#showyoursupport)
+
+- 🙏 [Acknowledgements](#acknowledgements)
+
+- ❓ [FAQ (OPTIONAL)](#faq)
+
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [portofolio] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[portofolio]** is my personal portofolio.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Tech stacks used on the project are.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">html</a></li>
+    <li><a href="#">css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Simple Hello Microverse</summary>
+  <ul>
+    
+  </ul>
+</details>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+### Install
+
+Install this project with:
+npm i
+
+### Usage
+
+To run the project, open index.html
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author**
+
+- GitHub: [@Bleu-Yves-Sopoude](https://github.com/Bleu-Yves-Sopoude)
+- Linkedin: [@Bleu Yves Sopoude](https://www.linkedin.com/in/bleu-yves/)
+
+
+## 🔭 Future Features<a name="future features"></a>
+
+    Describe 1 - 3 features you will add to the project.
+
+    [Linters]
+    [Html page]
+    [Css]
+
+  <p align="right"> (<a href="#readme-top"">back to top</a>) </p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+
+
+<p align="right">(<a href="#readme-top"">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="show your support"></a>
+
+   
+
+If you like this project support it by giving one star
+
+<p align="right">(<a href="#readme-top"">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgments"></a>
+
+  Give credit to everyone who inspired your codebase.
+
+I would like to thank my Microverse for this experience
+I would like to thank my code reviewer  shella12
+
+<p align="right">(<a href="#readme-top"">back to top</a>)</p>
+
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+    Add at least 2 questions new developers would ask when they decide to use your project.
+
+    [Question_1]
+        [Answer_1]
+
+    [Question_2]
+        [Answer_2]
+
+<p align="right">(<a href="#readme-top"">back to top</a>)</p>
```

Comment:
[OPTIONAL] Just like this comment 😉 the `FAQ` is optional, so if you're considering having one make sure to remove the hardcoded content on it, if not you may remove this section here and from the `Table of Contents` ❌